src/util/config.cpp

```diff
@@ -27,6 +27,7 @@ void SystemConfig::initialise()
     cgroupMode = getEnvVar("CGROUP_MODE", "on");
     netNsMode = getEnvVar("NETNS_MODE", "off");
     logLevel = getEnvVar("LOG_LEVEL", "info");
+    logFile = getEnvVar("LOG_FILE", "on");
```

Comment:
Policy-wise, I've opted for:
* 1 single log-level specified in `conf.logLevel`
* If an additional env variable (`conf.logFile`) is set, then logging also goes to a specific file (with the same log-level).

Reply:
I don't think file logging should be on by default. In most execution environments the task manager will handle piping stderr/stdout to somewhere useful (e.g. `kubectl`/ or `supervisord`). More importantly, something like supervisor will handle log file rotation. Having this on by default will lead to huge log files on our dev machines that we'll have to manually clear up. It should only be switched on when we need low-level debugging.